client/view/BrowserStats/BrowsersStat.css

```diff
@@ -0,0 +1,122 @@
+.BrowsersStat__globalCoverageBar {
+  display: flex;
+  gap: 2px;
+  justify-content: flex-start;
+  width: 100%;
+  padding: 0;
+  overflow: hidden;
+  counter-reset: alpha 0;
+  background-color: var(--main-bg);
+  border-radius: 6px;
+}
+
+.BrowsersStat__globalCoverageBarItem {
+  box-sizing: border-box;
+  width: calc(var(--proportion, 0) * 1%);
+  overflow: hidden;
+  line-height: 32px;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  list-style: none;
+  counter-increment: alpha -1;
+  background-color: var(--accent-alternave);
+  opacity: var(--alpha, 0%);
+
+  /* TODO types for this variables */
+  --proportion: 0;
+  --alpha: 0;
```

Comment:
I am afraid of not having component isolation for them

Reply:
Why is it a problem?
This variable set in this block and used in this blok. You will have the same result with any other name.

Reply:
Custom property is visible not only in this block but also in all nested blocks. It could create problems if we will have the same properties in nested block.

Reply:
If we will have variable with the same name in the nested block, those declarations will win the cascade. There won't be a problem.

Reply:
Don’t really like the lack of component isolation, but let’s try this way